.github/workflows/ci.yml

```diff
@@ -459,17 +556,135 @@ jobs:
       with:
         jobs: ${{ toJSON(needs) }}
 
+  github-release-pull-request:
+    name: >-
+      Open a GitHub Pull Request for
+      ${{ needs.pre-setup.outputs.git-tag }}
+    if: fromJSON(needs.pre-setup.outputs.release-requested)
+    needs:
+    - check
+    - pre-setup  # transitive, for accessing settings
+
+    runs-on: Ubuntu-latest
+
+    environment: git-repo
+
+    outputs:
+      pull_request_url: ${{ steps.pr.outputs.pull_request_url }}
+
+    permissions:
+      pull-requests: write
+
+    steps:
+    - name: Fetch the src
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0  # To reach the common commit w/ the repo default branch
+        ref: ${{ github.event.inputs.release-commitish }}
+        ssh-key: ${{ secrets.DEPLOYMENT_KEY }}
+    - name: Setup git user as [bot]
+      # Refs:
+      # * https://github.community/t/github-actions-bot-email-address/17204/6
+      # * https://github.com/actions/checkout/issues/13#issuecomment-724415212
+      uses: fregante/setup-git-user@v1.0.1
+    - name: Fetch the GHA artifact with the version patch
+      uses: actions/download-artifact@v2
+      with:
+        name: npm-package-bump-patch
+
+    - name: Apply the version patch
+      run: git am ${{ needs.pre-setup.outputs.version-patch-name }}
+    - name: Drop the version patch file
+      run: rm -fv ${{ needs.pre-setup.outputs.version-patch-name }}
+
+    - name: >-
+        Create a local 'release/${{
+          needs.pre-setup.outputs.dist-version
+        }}' branch
+      run: >-
+        git checkout -b 'release/${{
+          needs.pre-setup.outputs.dist-version
+        }}'
+
+    - name: >-
+        Push the `release/${{
+            needs.pre-setup.outputs.dist-version
+        }}` release branch to GitHub
+      run: >-
+        git push --atomic origin
+        'release/${{ needs.pre-setup.outputs.dist-version }}'
+
+    - name: pull-request-action
+      id: pr
+      uses: vsoch/pull-request-action@1.0.18
+      env:
+        BRANCH_PREFIX: ''
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        PULL_REQUEST_BODY: >-
+          Automated metadata update with the just released version
+          ${{ needs.pre-setup.outputs.dist-version }}
+        PULL_REQUEST_BRANCH: ${{ github.event.repository.default_branch }}
+        PULL_REQUEST_FROM_BRANCH: >-
+          release/${{ needs.pre-setup.outputs.dist-version }}
+        PULL_REQUEST_TITLE: >-
+          Bump version to ${{ needs.pre-setup.outputs.git-tag }}
+    - name: Log the pull request details
+      run: |
+        echo "PR number: ${{ steps.pr.outputs.pull_request_number }}"
+        echo "PR URL: ${{ steps.pr.outputs.pull_request_url }}"
+
+    - name: Append the release changelog to the PR description
+      run: >-
+        gh pr edit
+        '${{ steps.pr.outputs.pull_request_url }}'
+        --body-file='${{
+            needs.pre-setup.outputs.changelog-draft-name
+        }}'
```

Comment:
I forgot to add an env var here and the test failed because of that. This will be fixed by https://github.com/ansible/ansible-language-server/pull/201.